presto-main/src/test/java/com/facebook/presto/security/TestAccessControlManager.java

```diff
@@ -198,7 +198,7 @@ private static ConnectorId registerBogusConnector(CatalogManager catalogManager,
                 connectorId,
                 connector,
                 createInformationSchemaConnectorId(connectorId),
-                new InformationSchemaConnector(catalogName, nodeManager, metadata, accessControl),
+                new InformationSchemaConnector(catalogName, nodeManager, metadata, accessControl, transactionManager),
```

Comment:
Looks like https://github.com/prestosql/presto/pull/90/commits/7baabd1f658a6eccfeec61101f61cbaf3c972d13 removed all the `transactionManager` related stuff.